### ISSUE TITLE
datepicker change event not working

### ISSUE THREAD

**Issue by muhlisatac** [abpframework/abp#18335]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Hi 
In my razer page i have a datepicker like below.
`<abp-date-picker asp-for="DateFilter" label="@L["Date"].Value" />`

in javascript I have events like below. 
` $('#DateFilter').on('apply.daterangepicker', function (ev, picker) {
        alert(1);
    }); `

` $('#DateFilter').on('change', function () {
    alert(1);
}); `

But none of them is working. Can anyone help? I already looked at the datepicker [documentation](https://www.daterangepicker.com/#events).

### Known Workarounds

`<abp-input asp-for="DateFilter" type="text" label="@L["Date"].Value" />`

`$('#DateFilter').daterangepicker({
        "singleDatePicker": true,
        "showDropdowns": true,
        "autoApply": true,
        "linkedCalendars": false,
        "showCustomRangeLabel": false
    }, function (start, end, label) {
    });`

    `$('#DateFilter').on('apply.daterangepicker', function (ev, picker) {
        alert(1);
    });`

### Version

7.4.2

### User Interface

MVC

### Database Provider

EF Core (Default)

### Tiered or separate authentication server

None (Default)

### Operation System

Windows (Default)

**Comment by muhlisatac** [comment]
By the way, the code below which created by abp suite also doesn't work either for datetime filters.
`$('#AdvancedFilterSection select').change(function() {
        dataTable.ajax.reloadEx();;
    });`

**Comment by salihozkara** [comment]
> By the way, the code below which created by abp suite doesn't work either. `$('#AdvancedFilterSection select').change(function() { dataTable.ajax.reloadEx();; });`

@EngincanV
